segmentation/utils/phenotype_spatial_analysis.py

```diff
@@ -0,0 +1,106 @@
+import pandas as pd
+import numpy as np
+import xarray as xr
+import matplotlib
+import seaborn as sns
+from segmentation.utils import spatial_analysis_utils
+import importlib
+importlib.reload(spatial_analysis_utils)
+
+# Erin's Data Inputs
+# all_patient_data = pd.read_csv("/Users/jaiveersingh/Desktop/granA_cellpheno_CS-asinh-norm_matlab_revised.csv")
+# pheno = pd.read_csv("/Users/jaiveersingh/Downloads/CellType_GlobalSpatialEnrichment/cellpheno_numkey.csv")
+# dist_mat = np.asarray(pd.read_csv("/Users/jaiveersingh/Documents/MATLAB/distancesMat5.csv", header=None))
+
+
+def visualize_z_scores(z, pheno_titles):
```

Comment:
Let's put this is a new file called visualize.py, at the same level as spatial_analysis.py